### ISSUE TITLE
Accessibility - iOS - Search Results Do Not Notify VoiceOver User How Many Results Found

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor#100]
While attempting a search with VoiceOver the application does not notify VoiceOver user how many search results were found.

**Comment by jfc3-dol** [comment]
The search results screen is still not informing the VoiceOver user that anything was found and if so, how many.

**Comment by jfc3-dol** [comment]
VoiceOver is now announcing who any countries it has found, but when I tap the "Search" key, it does not announce the number. So I'm going to keep the issue open.

**Comment by jfc3-dol** [comment]
@harpreetsdev 

Will have to check into see if this is still a problem.

**Comment by harpreetsdev** [comment]
@jfc3-dol 

You the man sir. Appreciate that.

@pbhatt17 @Abychalil
